assets/queries/ansible/aws/iam_password_mininum_length/query.rego

```diff
@@ -0,0 +1,48 @@
+package Cx
+
+CxPolicy [ result ] {
+  document = input.document[i]
+  tasks := getTasks(document)
+  firstPolicy = tasks[_]
+  policyBody = firstPolicy["community.aws.iam_password_policy"]
+  policyName = firstPolicy.name
+
+   not policyBody.min_pw_length
+
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("name={{%s}}.{{community.aws.iam_password_policy}}", [policyName]),
+                "issueType":		"MissingAttribute",
+                "keyExpectedValue": "'min_pw_length' is set and no less than 8",
+                "keyActualValue": 	"'min_pw_length' is undefined"
+              }
+}
+
+CxPolicy [ result ] {
+  document = input.document[i]
+  tasks := getTasks(document)
+  firstPolicy = tasks[_]
+  policyBody = firstPolicy["community.aws.iam_password_policy"]
+  policyName = firstPolicy.name
+
+
+  to_number(policyBody.min_pw_length) < 8
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("name={{%s}}.{{community.aws.iam_password_policy}}.{{min_pw_length}}", [policyName]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": "'min_pw_length' is set and no less than 8",
+                "keyActualValue": 	"'min_pw_length' is less than 8"
+              }
+}
```

Comment:
The attribute 'min_pw_length' has an alias 'minimum_password_length'. Change these queries or add new ones to use when the alias is present.